src/js/components/Data/__tests__/__snapshots__/Data-test.tsx.snap

```diff
@@ -400,7 +400,7 @@ exports[`Data controlled search 1`] = `
     <span
       class="c11"
     >
-      4 results of undefined items
+      4 results of  items
```

Comment:
This read a little odd. It feels like ideally if the total wasn't provided it'd read either:
- "4 results" OR
- "4 results of unknown total items" (or something along those lines)

@MikeKingdom you're more expert in localization -- is this something we should worry about or accept this change?

Reply:
Yeah. In general we should never put "undefined" or "null" in a message. But we should also make sure we're not creating a message that isn't "friendly". Typically in this situation the guidance is if you don't know the total then only say "4 results".